### ISSUE TITLE
This function won't work as intended

### ISSUE THREAD

**Issue by KhalilKhunji** [SHAIKHAALMUTAWA/js-functions-lab#1]
https://github.com/SHAIKHAALMUTAWA/js-functions-lab/blob/57c6efd54c54781ddae9f35924dccc9ddddb3ea4/script.js#L64

You correctly identified that you needed the || (OR) operator - good! But if you tried to test this function with different arguments, you would find that it would always return true, regardless of what you put in it. Now why is that?

It's because [logical OR ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR) returns the the first truthy value if any of the operands (which in this case, are `letter ==='a'` `'e'` `'i'` `'u'` `'o'`)  is truthy, and if you recall, in Javascript, most things are truthy, with the exceptions being the following falsy:

- `null`
- `undefined`
- `NaN` (Not a Number)
- `0`
- Empty string ('', "", or ``)

Everything other than these is going to be truthy - and as you can see, `'e'` `'i'` `'u'` `'o'` are all non-empty strings, so they're truthy!

In other words, this function will always return true, because the condition you set for it guarantees that the OR operator will return a truthy value. Now, you were on the right track, all you had to do to get this properly working was:

```
if (letter ==='a' || letter ==='e' || letter ==='i' || letter ==='o' || letter ==='u') { 
```

Keep this in mind for the future!
